website/i18n/zh/docusaurus-plugin-content-blog/2022/06/14/automated-operation-base-apache-apisix.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "基于 Apache APISIX 的自动化运维平台"
+authors:
+  - name: "陈庆"
+    title: "Author"
+  - name: "韩飞"
+    title: "Technical Writer"
+    url: "https://github.com/hf400159"
+    image_url: "https://github.com/hf400159.png"
+keywords: 
+- Apache APISIX
+- API Gateway

```

Comment:
Please extract some scene keywords from the content, it helps Google to show results :) Like people may search `How to use APISIX with OIDC?`

Reply:
done, have checked every document